dump/dump.c

```diff
@@ -90,6 +90,12 @@ static int exfat_show_ondisk_all_info(struct exfat_blk_dev *bd)
 		goto free_ppbr;
 	}
 
+	if (memcmp(ppbr->bpb.oem_name, "EXFAT   ", 8) != 0) {
+		exfat_err("exfat filesystem is not found\n");
```

Comment:
How about this error message ?
"Bad fs_name in boot sector, which does not describe a valid exfat filesystem"

Reply:
Okay, it looks better.